<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitYearForm_template</class>
 <widget class="QDialog" name="SplitYearForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Split year dialog</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="Q3GroupBox" name="categoriesGroupBox">
       <property name="title">
        <string>Number of categories (you can select 1-3 categories)</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QSpinBox" name="categoriesSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="splitYearTextLabel">
     <property name="text">
      <string>Splitting year:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Q3GroupBox" name="category3GroupBox">
     <property name="title">
      <string>Category 3 (you can select 2-6 divisions)</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QSpinBox" name="category3SpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category3Division1LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category3Division2LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category3Division3LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category3Division4LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category3Division5LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category3Division6LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton3">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton4">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <spacer name="spacer6_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>110</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <spacer name="spacer3_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="Q3GroupBox" name="category2GroupBox">
     <property name="title">
      <string>Category 2 (you can select 2-6 divisions)</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QSpinBox" name="category2SpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category2Division1LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category2Division2LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category2Division3LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category2Division4LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category2Division5LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category2Division6LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="Q3GroupBox" name="groupBox5">
     <property name="title">
      <string>Separator character(s)</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="separatorLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="spacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Q3GroupBox" name="category1GroupBox">
     <property name="title">
      <string>Category 1 (you can select 2-12 divisions)</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QSpinBox" name="category1SpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="category1Division1LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="category1Division2LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="category1Division3LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="category1Division4LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="category1Division5LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="category1Division6LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="category1Division7LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLineEdit" name="category1Division8LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLineEdit" name="category1Division9LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLineEdit" name="category1Division10LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLineEdit" name="category1Division11LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLineEdit" name="category1Division12LineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>categoriesSpinBox</tabstop>
  <tabstop>category1SpinBox</tabstop>
  <tabstop>category1Division1LineEdit</tabstop>
  <tabstop>category1Division2LineEdit</tabstop>
  <tabstop>category1Division3LineEdit</tabstop>
  <tabstop>category1Division4LineEdit</tabstop>
  <tabstop>category1Division5LineEdit</tabstop>
  <tabstop>category1Division6LineEdit</tabstop>
  <tabstop>category1Division7LineEdit</tabstop>
  <tabstop>category1Division8LineEdit</tabstop>
  <tabstop>category1Division9LineEdit</tabstop>
  <tabstop>category1Division10LineEdit</tabstop>
  <tabstop>category1Division11LineEdit</tabstop>
  <tabstop>category1Division12LineEdit</tabstop>
  <tabstop>category2SpinBox</tabstop>
  <tabstop>category2Division1LineEdit</tabstop>
  <tabstop>category2Division2LineEdit</tabstop>
  <tabstop>category2Division3LineEdit</tabstop>
  <tabstop>category2Division4LineEdit</tabstop>
  <tabstop>category2Division5LineEdit</tabstop>
  <tabstop>category2Division6LineEdit</tabstop>
  <tabstop>category3SpinBox</tabstop>
  <tabstop>category3Division1LineEdit</tabstop>
  <tabstop>category3Division2LineEdit</tabstop>
  <tabstop>category3Division3LineEdit</tabstop>
  <tabstop>category3Division4LineEdit</tabstop>
  <tabstop>category3Division5LineEdit</tabstop>
  <tabstop>category3Division6LineEdit</tabstop>
  <tabstop>separatorLineEdit</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>pushButton3</tabstop>
  <tabstop>pushButton4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
